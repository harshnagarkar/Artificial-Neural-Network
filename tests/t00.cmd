 toy_trainInput.txt toy_trainOutput.txt toy_testInput.txt toy_testOutput.txt toy_structure.txt toy_weights.txt 1
