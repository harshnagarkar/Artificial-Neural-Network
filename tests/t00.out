0.300167051381 0.199995216844 
2
2
2
2
0.250000000000
